include/RbtAlignTransform.h

```diff
@@ -27,9 +27,22 @@ class RbtAlignTransform: public RbtBaseBiMolTransform {
     static RbtString _COM;
     static RbtString _AXES;
 
+    enum LigandCenterOfMassPlacementStrategy { COM_ALIGN, COM_RANDOM };
+    enum LigandAxesAlignmentStrategy { AXES_ALIGN, AXES_RANDOM };
+
+    struct Config {
+      LigandCenterOfMassPlacementStrategy center_of_mass_placement_strategy;
+      LigandAxesAlignmentStrategy axes_alignment_strategy;
+    };
+
+    static constexpr Config DEFAULT_CONFIG {
+      .center_of_mass_placement_strategy = LigandCenterOfMassPlacementStrategy::COM_ALIGN,
+      .axes_alignment_strategy = LigandAxesAlignmentStrategy::AXES_ALIGN,
+    };
```

Comment:
after the previous comment's changes, if we want to keep a default config as constexpr, it would look like this:
```suggestion
    static constexpr Config DEFAULT_CONFIG;
```

Reply:
Removed the `constexpr` as it was the result of me not reading the errors properly. I thought I couldn't define a regular const but what the error was actually saying is that I should initialize it outside of the class declaration. Fixed now.